xmodule/js/spec/time_spec.js

```diff
@@ -1,7 +1,7 @@
 
 'use strict';
 
-import * as Time from 'time.js';
+import * as Time from '../src/time.js';
```

Comment:
Here I had to made correction in path and similar corrections in some other files. Again I don't understand how is it working on master?